Dockerfile

```
@@ -176,8 +181,9 @@ RUN apt-get install -y tzdata
 RUN set -eux; \
     packages=""; \
     for pg in ${PG_VERSIONS}; do \
-        packages="$packages postgresql-${pg} postgresql-server-dev-${pg} postgresql-${pg}-dbgsym \
-            postgresql-plpython3-${pg} postgresql-plperl-${pg} postgresql-${pg}-pgextwlist postgresql-${pg}-hll \
+        export FULL_VERSION="$(/build/scripts/pg_version.sh ${pg})*" ; \
```

why do you need `export` here?
```suggestion
        FULL_VERSION="$(/build/scripts/pg_version.sh ${pg})*" ; \
```